### PR TITLE
servicebroker: Change the order to invoke express middleware

### PR DESCRIPTION
**Pull request by cdlliuy**
[#121943583]

### CODE REVIEW

**Comment by cdlliuy** [comment]
Also, the PR is updated to include the feature for "provision service"

**Comment by fraenkel** [comment]
Next time split the PRs. It is confusing when you deliver multiple stories.

File: servicebroker/config/settings.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "port": 8080,
   "user": "user",
```
Comment (fraenkel) [inline servicebroker/config/settings.json]: For consistency, let us name this username instead of user.

**Comment by fraenkel** [comment]
We need to have one style of code. I don't care which. Please look at how the API component used Sequelize. There may be a combination of both which is best but right now we have two different ways of doing similar DB operations.

File: servicebroker/config/settings.json (hunk 1 above)
Comment (cdlliuy) [inline servicebroker/config/settings.json]: Michael, how about this time? I see the branch has conflicts now. Should I raise another 2 PR instead?

Comment (fraenkel) [inline servicebroker/config/settings.json]: You should pull in the latest develop and rebase your branch. The rebase will probably have a conflict which should force you to fix things. At which point once you commit the rebase you should be good with a PR.

Comment (cdlliuy) [inline servicebroker/config/settings.json]: ok, will do the rebase, and also move the createService part out of the PR.   For create Service part,  I will raise another PR, and change the style for Sequelize call

**Comment by fraenkel** [comment]
Merged the reorder commit.

**Pull request closed without merging**